src/perfect-number.js

```diff
@@ -0,0 +1,12 @@
+function isPerfect(n) {
+  let sum = 0;
+  for (let i = 1; i <= n / 2; i += 1) {
+    if (n % i === 0) sum += i;
+  }
+  if (sum === n) return true;
```

Comment:
remove lines 6,7. return true instead

Reply:
If sum of divisors is equal to n, then n is a perfect number.